packages/auth/amplify_auth_cognito/example/lib/main.dart

```diff
@@ -54,20 +54,27 @@ class _MyAppState extends State<MyApp> {
     GoRoute(
       path: '/update-user-attribute',
       builder: (BuildContext _, GoRouterState state) =>
-          const UpdateUserAttributeScreen(),
+          UpdateUserAttributeScreen(
```

Comment:
Note: The updates here were just to make testing easier or fix broken routes. They aren't related to the changes in the PR.